### PR TITLE
gulp version bug with the babel register module require

### PR DESCRIPTION
**Pull request by JoshK2**
gulp do a require to babel register module and failed to load it:
Failed to load external module @babel/register
if you downgrade the version of gulp from 3.9.1 to 3.9.0 its working good.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "del": "^3.0.0",
     "eslint": "^4.7.2",
     "grommet-toolbox": "^2.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^3.9.0",
```
Comment (alansouzati) [inline package.json]: this will install `3.9.1` right ? since we have the `^` there.

Comment (JoshK2) [reply package.json]: Yes, but thus version is failed to load the babel module

Comment (alansouzati) [reply package.json]: yes I get it. but this PR does not fix this problem since it will install 3.9.1 still right ?

Comment (alansouzati) [reply package.json]: can you get me more details on how to reproduce this issue?

**Comment by ShimiSun** [comment]
Hi @JoshK2, thanks for the PR. will you be willing to address the conflicts?

**Comment by JoshK2** [comment]
> will you be willing to address the conflicts?

The solution that I think worked, seems to not working.
So I don't know how to solve the problem, and I think is not important because I update to version 1.13.0 to use the "hold" grommet and everything working.
You can close the pull request.

**Pull request closed without merging**